+++ src/lib/fuse-ext2/e2fsprogs-1.41.12.newgit/ext2fs/ext2fs.h
@@ -49,9 +49,12 @@
 #include <sys/types.h>
 #endif
 
+typedef unsigned int dev_t;
+
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <time.h>
 
 #if EXT2_FLAT_INCLUDES
 #include "e2_types.h"
